Annotation/constituent_golden_set.json

```diff
@@ -232,14 +227,26 @@
                     }
                 ],
                 "type": "constituent",
-                "clause": "",
+                "clause": "how DC might collect the various crossover miniseries specials",

```

Comment:
I think the clause should include ", however, perhaps along with the lead story from the Infinite Crisis Secret Files"